apis/Google.Cloud.Language.V1/Google.Cloud.Language.V1/LanguageServiceClient.cs

```diff
@@ -58,6 +58,11 @@ private LanguageServiceSettings(LanguageServiceSettings existing) : base(existin
             OnCopy(existing);
         }
 
+        /// <summary>
+        /// This will be in GAX...
+        /// </summary>
+        public grpccore::Interceptors.Interceptor Interceptor { get; set; }
```

Comment:
If the setter is public here, does it mean that a user can change the interceptor after using Create(channel, settings);

Reply:
Yes, but it won't have any impact. That's true of most (all?) service-wide settings.